### PR TITLE
Update API version, update data contract for Order APIs

### PR DESCRIPTION
**Pull request by anponnet**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by anponnet** [comment]
Hi @sarangan12 Will you be able to review it today?

**Comment by sarangan12** [comment]
@anponnet I am a little confused. When you update an API version, I expect the older version files to be there. In your case, you have actually removed them, Are you retiring them?

**Comment by anponnet** [comment]
> confus



> @anponnet I am a little confused. When you update an API version, I expect the older version files to be there. In your case, you have actually removed them, Are you retiring them?

@sarangan12 Yes, we are still not in GA. so we wanted to have a newer version in GA

File: specification/edgegateway/resource-manager/readme.ruby.md [hunk 1]
```diff
@@ -12,15 +12,15 @@ azure-arm: true
...
+### Tag: package-2019-03 and ruby
 
-These settings apply only when `--tag=package-2018-07 --ruby` is specified on the command line.
+These settings apply only when `--tag=package-2019-03 --ruby` is specified on the command line.
 Please also specify `--ruby-sdks-folder=<path to the root directory of your azure-sdk-for-ruby clone>`.
 
-``` yaml $(tag) == 'package-2018-07' && $(ruby)
-namespace: "Azure::Compute::Mgmt::V2018_07_01"
+``` yaml $(tag) == 'package-2019-03' && $(ruby)
+namespace: "Azure::Compute::Mgmt::V2019_03_01"
```
Comment (sarangan12) [inline specification/edgegateway/resource-manager/readme.ruby.md]: Your namespace cannot be Azure::Compute.  (You are not Compute Service r8???) It must be Azure::EdgeGateway.

**Review by sarangan12 — CHANGES_REQUESTED** [review]

File: specification/edgegateway/resource-manager/readme.ruby.md (hunk 1 above)
Comment (anponnet) [reply specification/edgegateway/resource-manager/readme.ruby.md]: Fixed

**Review by sarangan12 — APPROVED** [review]

**Pull request merged**